live-examples/js-examples/string/string-raw.html

```diff
@@ -0,0 +1,9 @@
+<pre>
+<code id="static-js">const name = 'Bob';
+
+// The next line returns the template with the
+// value of the name variable inserted.
+console.log(string.raw`<p>Good morning, {name}</p>`);
+</code>
+</pre>
+    
```

Comment:
Please remove the spaces from this line, and all files should end with a new line.